src/shared/modules/itemEditor/itemEditorDuck.js

```diff
@@ -89,9 +89,7 @@ export const handleFetchDataEpic = (action$, store) =>
         return noop
       })
```

Comment:
now this file is hard to review cause of formatting changes. be on the same formatting